themes-default/slim/tsconfig.json

```diff
@@ -4,7 +4,8 @@
         "noImplicitAny": true
     },
     "include": [
-        "./static/js/*.js"
+        "./static/js/*.js",
+        "./src/**/*.{vue|js}"
```

Comment:
Could be wrong, but isn't the syntax `./src/**/*.{vue,js}` - comma instead of a pipe?